hooks/useTutorData.ts

```
@@ -23,7 +25,8 @@ export function useFetchTutorData() {
         validate?: boolean;
       }
     }) => {
-      return trpc.tutorData.fetch.mutate({ url, options });
+      // Use type assertion to avoid strict inference issues on the app side
+      return (trpc as any).tutorData.fetch.mutate({ url, options }) as Promise<inferRouterOutputs<AppRouter>["tutorData"]["fetch"]>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)